src/App.tsx

```diff
@@ -1,24 +1,91 @@
-import React from 'react';
+import React, { Props } from 'react';
 import './App.scss';
+import Clock from './components/Clock';
 
-const App: React.FC = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type State = {
+  isClockVisible: boolean;
+  clockName: number;
 };
 
+class App extends React.Component<Props<State>, State> {
+  state = {
+    isClockVisible: false,
```

Comment:
```suggestion
    isClockVisible: true,
```
Clock should be visible by default so you can set it in the state to `true` and there's no need in componentDidMount here